src/test/HomePage.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import { BrowserRouter as Router } from 'react-router-dom'
+import HandleError from 'utils/HandleError'
+import Content from 'Components/Content'
+import HomePage from 'Pages/HomePage'
+
+describe('Fetch and render from the API', () => {
+    const HomePageStructure =
+    <Router>
+        <HandleError message={"We're sorry, something went wrong"}>
+            <Content>
+                <HomePage>
+                </HomePage>
+            </Content>
+        </HandleError>
+    </Router>
```

Comment:
You should be able to acomplish this using the custom render from react testing library. Let me share it with you: https://testing-library.com/docs/react-testing-library/setup/#custom-render

Also remember that if the component doesn't have children it should be self-closing: <HomePage />